Npgsql.sln

```diff
@@ -47,6 +47,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Github", "Github", "{BA7B6F
 		.github\workflows\rich-code-nav.yml = .github\workflows\rich-code-nav.yml
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Npgsql.OpenTelemetry.Tests", "test\Npgsql.OpenTelemetry.Tests\Npgsql.OpenTelemetry.Tests.csproj", "{44E4A08B-3D46-43CF-B030-8501F98EA1D7}"
```

Comment:
Any reason to have a special test project for OpenTelemetry, rather than just putting the tests inside Npgsql.Tests?

Reply:
Not particularly, I just noticed that there was a specific one for NodaTime, so thought that was the convention.
I'm happy to move to the general test project if you'd prefer?

Reply:
NodaTime was split out because it interfere with global type mappings and stuff. Yeah, better to keep it in the same Npgsql.Tests for simplicity unless there's a specific reason to do otherwise.